### DISCUSSION TITLE
글자가 UI 밖으로 넘어갑니다

### DISCUSSION THREAD

**Discussion by OABHs** [wb1016/sb_korpatch#42] [General]
![이슈1](https://user-images.githubusercontent.com/66473940/124379888-a5fec400-dcf4-11eb-9185-e19e2ae24a48.png)

'차 마실땐 절대 노바키드를 초대하지 마(Never Invite a Novakid to Tea)' 코덱스의 경우 아이템 창에 있는 해당 코덱스 아이템에 커서를 올렸을 시 나타나는 인터페이스에서 글자가 UI를 뚫고 나갑니다.

![이슈2](https://user-images.githubusercontent.com/66473940/124379890-a72ff100-dcf4-11eb-83d6-49f45f07ec4e.png)
![이슈3](https://user-images.githubusercontent.com/66473940/124379892-a72ff100-dcf4-11eb-818f-123db49f1cde.png)

'고대 문자(The Ancient Alphabet)' 코덱스의 경우 내용을 건들지 않았는데도 저렇게 깨집니다. 아마도 폰트가 문제되는 것 같은데, 각 알파벳에 대응하는 고대 문자들도 바닐라와 달라지는 것 같습니다.

**Comment by OABHs** [comment]
두번째 문제는 아마도 고대어 폰트를 건드리는 이미 없는 모드를 잘못 인식해서 생겨난 것으로 스타바운드 재설치후 해결되었습니다. 한글패치에는 전혀 문제가 없습니다.

**Comment by wb1016** [comment]
issue로 이동했습니다.
